ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -268,15 +268,7 @@ interface InstitutionEditState {
   apiErrorMsg: string;
   institutionMode: InstitutionMode;
   institution: Institution;
-  institutionToEdit: Institution;
-  invalidRtEmailAddress: boolean;
-  invalidRtEmailAddressMsg: string;
-  invalidCtEmailAddress: boolean;
-  invalidCtEmailAddressMsg: string;
-  invalidRtEmailDomain: boolean;
-  invalidRtEmailDomainsMsg: string;
-  invalidCtEmailDomain: boolean;
-  invalidCtEmailDomainsMsg: string;
+  institutionBeforeEdits: Institution;
```

Comment:
This is the original Institution before any editing.  I renamed it because I thought it was clearer.  LMKWYT